backstage/app-config.yaml

```diff
@@ -111,13 +111,22 @@ auth:
           resolvers:
             - resolver: emailMatchingUserEntityProfileEmail
 
-    github:
-      development:
-        clientId: ${AUTH_GITHUB_CLIENT_ID}
-        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
-      production:
-        clientId: ${AUTH_GITHUB_CLIENT_ID}
-        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
+    # github:
```

Comment:
I don't think we should have commented-out code in main. If there's a reason it's here, I think it could be accompanied by an explanation, e.g. "uncomment this to do X alternate functionality"